ecosystem/sep-0024.md

```diff
@@ -142,7 +142,7 @@ Name | Type | Description
 -----|------|------------
 `asset_code` | string | The code of the stellar asset the user wants to receive for their deposit with the anchor.  If the user wants to receive lumens use the code `XLM`.
 `asset_issuer` | string | (optional) The issuer of the stellar asset the user wants to receive for their deposit with the anchor.  If the user wants to receive lumens use the issuer `native`.  If asset_issuer is not provided, the anchor should use the asset issued by themselves as described in their TOML file.
-`amount` | number | (optional) Amount of asset requested to deposit.  If this is not provided it will be collected in the interactive flow.
+`amount` | number | (optional) Amount of asset requested to deposit.  If this is not provided it will be collected in the interactive flow.  If the user wants to receive lumens this argument should be ignored.
```

Comment:
`amount` should be ignored if `asset_code` is `XLM` because the amount of XLM deposited to the user's stellar account is dependent on the amount sent to the anchor's non-stellar account and the rate of exchange between XLM and the non-stellar asset.